_includes/macros.html

```diff
@@ -238,3 +238,50 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">Share this page</h1>
         <p class="alert_text">{{ text }}</p>
     </div>
 {%- endmacro %}
+
+
+
+{# ==========================================================================
+
+   subscribe()
+
+   ==========================================================================
+
+   Description:
+
+   Creates an email subscription form when given:
+
+   code: A GovDelivery code a specified mailing list
+
+   ========================================================================== #}
+
+{% macro subscribe(code) %}
+
+<form id="email-subscribe-form"
+      action="/subscriptions/new/"
+      method="POST"
+      enctype="application/x-www-form-urlencoded">
+
+    <div class="form-group">
+        <label for="email-subscribe-form_email">
+            Email address
+        </label>
+        <input id="email-subscribe-form_email"
+               type="email"
+               name="email"
+               placeholder="example@mail.com">
+    </div>
+    <p class="short-desc u-mt10">
+        The information you provide will permit the Consumer Financial
+        Protection Bureau to process your request or inquiry.
+        <a href="/privacy-policy">See More</a>
```

Comment:
We do prefer our URLs with trailing slashes. In fact, they are added automatically, if missing.